taxi/tests/test_models.py

```diff
@@ -0,0 +1,57 @@
+from django.test import TestCase
+
+from taxi.models import Driver, Car, Manufacturer
+
+
+class DriverModelTests(TestCase):
+    @classmethod
+    def setUpTestData(cls) -> None:
+        Driver.objects.create(
+            username="test",
+            password="test123",
```

Comment:
The `password` field should be hashed when creating a `Driver` instance. Consider using Django's `set_password` method to ensure the password is stored securely.